src/bond.py

```diff
@@ -3,3 +3,65 @@
 # Part of azplugins, released under the BSD 3-Clause License.
 
 """Bond potentials."""
+
+from hoomd.azplugins import _azplugins
+from hoomd.data.parameterdicts import TypeParameterDict
+from hoomd.data.typeparam import TypeParameter
+from hoomd.md import bond
+
+
+class _Bond(bond.Bond):
+    _ext_module = _azplugins
+
+
+class DoubleWell(_Bond):
+    r"""Double-well bond potential.
+
+    `DoubleWell` specifies a double well potential between the two particles in
+    the simulation state with:
+
+    .. math::
+
+        U(r)  =  \frac{V_{max}-c/2}{b^4} \left[
+            \left( r - a/2 \right)^2 - b^2 \right]^2
+            +\frac{c}{2b}(r - a/2) + c/2
+
+    Attributes:
```

Comment:
Documentation needs to be updated. Same comment as above, can be left for later but we should note that somewhere with a TODO.